### ISSUE TITLE
PanelEdit: ColorPickerPopover cannot be opened when Enter key is pressed.

### ISSUE THREAD

**Issue by thisisobate** [grafana/grafana#42526]
**Steps to reproduce**:
Create a timeSeries panel and navigate to the panelEdit. In panelEdit, navigate to the thresholds section. You'll find a color picker (circled in red). Try opening it by pressing `Enter` key. 
**Actual Result**:
ColorPickerPopover is not opened.
**Expected Result**
ColorPickerPopover is opened.
**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: Grafana v8.4.0-pre (f454a5ce60)
- Data source type & version:
- User OS & Browser: MacOs, Chrome
- Others:

**Comment by thisisobate** [comment]
duplicate of #42525
